tests/SingleValueObjectTest.php

```diff
@@ -14,16 +13,16 @@
 class SingleValueObjectTest extends TestCase
 {
     /** @test */
-    public function single_property_value_object_can_be_created_with_default_value()
+    public function single_property_value_object_can_be_created_with_default_value(): void
     {
         $subject = new SingleVO();
 
         $this->assertInstanceOf(SingleValueObject::class, $subject);
-        $this->assertEquals(null, $subject->value);
+        $this->assertNull($subject->value);
```

Comment:
I usually prefer having the `assertEquals` method with an explicit `null` value instead of the `assertNull` method. Again, maybe in this specific case seems overkill, but I prefer to be able to make changes that in git seem more concise (instead of changing the whole method I just change the value).

Here, it happens to be the case that the default value is defined as being null, but what if later on I decide that it should be an `'empty'` string instead? (I know it sounds silly, but bear with me) I just need to change the value and in the change history, it would make my intention a bit clearer.

This is my preferred choice and quirks, there's nothing wrong with either way of doing it.